### DISCUSSION TITLE
Can using Fusion Auth for Amplication?

### DISCUSSION THREAD

**Discussion by nguyenthanhtien** [amplication/amplication#8289] [Q&A]
Our team wanna using FusionAuth with Amplication, how to add OIDC plugins support to integrate with Amplication?

**Comment by nguyenthanhtien** [comment]
Anyone here?

**Reply by overbit** [reply]
Hi @nguyenthanhtien, as this is not in the current plugin catalog, we can definitely add support for a generic OIDC plugin if FusionAuth support it. 
I'd like to have a chat to better understand your case, would you available for a chat?

**Reply by nguyenthanhtien** [reply]
@overbit I also make question to FusionAuth. But i dont have experience with OIDC and firsttime using NestJS. You can reference my question to fusionauth team. They answer and let me know need integrate with OIDC. I dont know currently amplication support or not, can guide for me how to integrate with amplication?
Sub-question: when amplication release Build with .NET? i want try using that.

> I'd like to have a chat to better understand your case, would you available for a chat?

Which channel you available for chat, please send me?

https://github.com/FusionAuth/fusionauth-issues/issues/2703#event-12310796095

**Reply by overbit** [reply]
@nguyenthanhtien as I can see from the comment, they support any OIDC integrations. 
Then it's just a matter of having a new plugin for it. 
As we currently support official and community plugins, you can implement a new plugin for it so you would able to use it in amplication. In case you need support from amplication, I'd suggest to reach out to us so we can organise a call :)

**Reply by nguyenthanhtien** [reply]
> @nguyenthanhtien as I can see from the comment, they support any OIDC integrations. Then it's just a matter of having a new plugin for it. As we currently support official and community plugins, you can implement a new plugin for it so you would able to use it in amplication. In case you need support from amplication, I'd suggest to reach out to us so we can organise a call :)

Sorry but i dont have exp with OIDC. Which plugin office support for OIDC? Can we using which plugin for OIDC integration?

**Reply by nguyenthanhtien** [reply]
Sub-question: when amplication release Build with .NET? i want try using that.

**Reply by overbit** [reply]
> Sub-question: when amplication release Build with .NET? i want try using that.

for the .NET early access you can submit your interest here: https://amplication.com/dot-net-early-access

**Reply by nguyenthanhtien** [reply]
> > Sub-question: when amplication release Build with .NET? i want try using that.
> 
> for the .NET early access you can submit your interest here: https://amplication.com/dot-net-early-access

thanks @overbit, i just request try .NET early access.
